src/source/Signaling/LwsApiCalls.h

```diff
@@ -249,6 +250,7 @@ STATUS writeLwsData(PSignalingClient, BOOL);
 STATUS terminateLwsListenerLoop(PSignalingClient);
 STATUS receiveLwsMessage(PSignalingClient, PCHAR, UINT32);
 STATUS getMessageTypeFromString(PCHAR, UINT32, SIGNALING_MESSAGE_TYPE*);
+PCHAR getMessageTypeInString(SIGNALING_MESSAGE_TYPE);
```

Comment:
It seems like the other APIs fill a user allocated buffer and return a length maybe we should do that for consistency?  Not really a big deal it looks like the use case for this method is only for printing so definitely a lot more "convenient" to return PCHAR as you have done.

Reply:
Yes. You are correct, since it is purely for logging purposes, I added a very simple function without any user allocated buffer.